Makefile

```diff
@@ -509,6 +509,14 @@ format-promql-tests:
 check-promql-tests: format-promql-tests
 	@./tools/find-diff-or-untracked.sh $(PROMQL_TESTS) || (echo "Please format PromQL test files by running 'format-promql-tests'" && false)
 
+.PHONY: format-protobuf
+format-protobuf:
```

Comment:
can we make this execute after someone runs `make protos`?

Reply:
It would certainly be possible, but I'd personally find that behaviour a bit surprising (eg. imagine if building the binary also formatted the Go code).